src/routes/ExercisesScreens.tsx

```diff
@@ -26,39 +25,6 @@ const ItemTitle = styled(FlatList)`
   padding-left: ${wp('5%')}px;
 ` as ComponentType as new () => FlatList<Exercise>
 
-const Description = styled.Text<{ selected: boolean }>`
-  font-size: ${props => props.theme.fonts.defaultFontSize};
-  font-family: ${props => props.theme.fonts.contentFontRegular};
-  font-weight: ${props => props.theme.fonts.lightFontWeight};
-  color: ${props => (props.selected ? props.theme.colors.white : props.theme.colors.lunesGreyDark)};
-`
-
-const Container = styled.Pressable<{ selected: boolean }>`
-  align-self: center;
-  padding: 17px 8px 17px 16px;
```

Comment:
All list items are using `padding: 12px 8px 12px 16px;`now, I am open to changing this to one or another though.